app/controllers/chargeback_assignment_controller.rb

```diff
@@ -4,110 +4,79 @@ class ChargebackAssignmentController < ApplicationController
   after_action :cleanup_action
   after_action :set_session_data
 
-  include Mixins::SavedReportPaging
   include Mixins::GenericSessionMixin
   include Mixins::BreadcrumbsMixin
 
   def self.table_name
     @table_name ||= "chargeback_assignment"
   end
 
-  def tree_select
-    self.x_active_tree = params[:tree] if params[:tree]
-    self.x_node = params[:id]
-    get_node_info(x_node)
-    replace_right_cell
+  def index
+    @title = _("Chargeback Assignments")
+    @tabform = "Compute"
+    session[:changed] = @changed = false
+    build_tabs
+    set_form_vars
+    render :action => "index"
   end
 
-  def explorer
-    @breadcrumbs = []
-    @explorer    = true
-    build_accordions_and_trees
-
-    @right_cell_text = _("All Assignments")
-    set_form_locals if @in_a_form
-    session[:changed] = false
-
-    render :layout => "application" unless request.xml_http_request?
-  end
-
-  def set_form_locals
-    @x_edit_buttons_locals = {
-      :action_url   => 'cb_assign_update',
-      :no_cancel    => true,
-      :multi_record => true
-    }
+  def change_tab
+    clear_flash_msg
+    @tabform = params['uib-tab']
+    build_tabs
+    set_form_vars
+    render :action => "index"
```

Comment:
This feels like a fullpage reload to me, can't we have an AJAX replace instead? cc @himdel

Reply:
Nevermind, all other pages have it the same way.